examples/auth/app/users/queries/getUser.ts

```diff
@@ -7,6 +7,7 @@ type GetUserInput = {
 
 export default async function getUser({where}: GetUserInput, ctx: Ctx) {
   ctx.session.$authorize()
+  ctx.session.$isAuthorized()
```

Comment:
These aren't working correctly. `$authorize()` is typed `...args: any[]`. 

`$isAuthorized()` picks up the correct type but has two overloads for `...args: any[]` which allows incorrect usage 🤔